source/cipher.c

```diff
@@ -88,6 +88,56 @@ static enum aws_cryptosdk_sha_version aws_cryptosdk_which_sha(enum aws_cryptosdk
     }
 }
 
+static bool str_eq(const char *const str1, const char *const str2) {
```

Comment:
Is this comparison constant time?

Reply:
I will remove these comparisons for now, and simplify the `aws_cryptosdk_alg_properties_is_valid` function, as these properties are not necessary for the `framefmt` harnesses. We will revisit and extend this validity function when we go over proof harnesses for `cipher.c`.